addons/mail/static/src/components/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection_tests.js

```diff
@@ -91,6 +91,56 @@ QUnit.test('select another mailbox', async function (assert) {
     );
 });
 
+QUnit.test('navigate to inbox tab when discuss is loaded from another tab than mailbox (like when user come from a notification)', async function (assert) {
+    assert.expect(5);
+
+    this.data['mail.channel'].records.push({ id: 20 });
+    this.data['mail.message'].records.push({
+        channel_ids: [20],
+        date: "2019-04-20 10:00:00",
+        id: 100,
+        model: 'mail.channel',
+        res_id: 20,
+    });
+
+    await this.start({
+        discuss: {
+            params: {
+                default_active_id: 20,
+            },
+        }
+    });
+
+    assert.hasClass(
+        document.querySelector('.o_MobileMessagingNavbar_tab[data-tab-id="channel"]'),
+        'o-active',
+        "the 'channel' tab should be the active tab"
+    );
+
+    await afterNextRender(() => document.querySelector('.o_MobileMessagingNavbar_tab[data-tab-id="mailbox"]').click());
+
+    assert.doesNotHaveClass(
+        document.querySelector('.o_widget_Discuss_controlPanelButtonMarkAllRead'),
+        'o_hidden',
+        "should contains Mark all read button on control panel"
+    );
+    assert.hasClass(
+        document.querySelector('.o_MobileMessagingNavbar_tab[data-tab-id="mailbox"]'),
+        'o-active',
+        "the 'mailboxes' tab should be the active tab"
+    );
+    assert.hasClass(
+        document.querySelector('.o_DiscussMobileMailboxSelection_button'),
+        'o-active',
+        "the 'inbox' inner-tab should be the active tab"
+    );
```

Comment:
Also in assertion comments after a user interaction, it's good to recall context of expectation.
For instance: _"'Inbox' mailbox should be auto-selected after click on mailbox tab."_

Reason is that assertion failure is on QUnit UI, and it's annoying to have to go in the code of test, take some time to read the test, and finally understand why the assertion fails (whether it's intentional or not, test badly written or actual code breaks this, etc.).